src/__tests__/starWarsIntrospection-test.ts

```diff
@@ -1,13 +1,11 @@
-// @flow strict
-
 import { expect } from 'chai';
 import { describe, it } from 'mocha';
 
-import { graphqlSync } from '../graphql';
+import { graphqlSync } from '../index';
```

Comment:
You shouldn't change imports these PR should contain only TS migration.
We also have a rule of not include `index` files in general.

Reply:
Oops, I blame VSCode! 🤣